README.md

```diff
@@ -1,5 +1,44 @@
 # amundsengremlin
 Amundsen Gremlin
```

Comment:
I think this bit of text makes the markdown rendering of the badge line look a little wonky, and is already captured in the h1 heading.
```suggestion
```